app/main.py

```diff
@@ -1 +1,47 @@
-#  write your code here
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
```

Comment:
fix indentation here

Reply:
sorry but how i have to do it. i don't understand. sorry again